rotkehlchen/data_import/importers/binance.py

```diff
@@ -187,7 +189,7 @@ def process_trades(
         grouped_trade_rows = []
         while len(rows_grouped_by_fee[False]) > 0:
             cur_batch = [rows_grouped_by_fee[False].pop(), rows_grouped_by_fee[False].pop(0)]
-            if True in rows_grouped_by_fee:
+            if True in rows_grouped_by_fee and len(rows_grouped_by_fee[True]) > 0:
```

Comment:
Hmm why is it needed?

Reply:
Because in the list of events saw a case of a trade where there was 3 trades and 2 fee items. In that case the pop here is for an event (the fee) that doesn't exists

Reply:
Does the execution really achieve that point? If yes, then I think there is a bug because if I remember correctly this case should be checked here and rejected
![image](https://user-images.githubusercontent.com/63492346/203988869-d60649a8-c4d2-4f24-8544-cd4fa78c326f.png)

Reply:
Yes, I didn't add a test for that but I should if is not a known branch of execution